src/Tribe/Admin/Move_Ticket_Types.php

```diff
@@ -0,0 +1,364 @@
+<?php
+class Tribe__Tickets__Admin__Move_Ticket_Types extends Tribe__Tickets__Admin__Move_Tickets {
+	protected $dialog_name = 'move_ticket_types';
+
+	public function __construct() {
+		parent::__construct();
+		add_action( 'wp_ajax_move_ticket_types_post_list', array( $this, 'update_post_choices' ) );
+		add_action( 'wp_ajax_move_ticket_type', array( $this, 'move_ticket_type_requests' ) );
+		add_action( 'tribe_tickets_ticket_type_moved', array( $this, 'notify_attendees' ), 100, 3 );
+		add_action( 'tribe_events_tickets_metabox_advanced', array( $this, 'expose_ticket_history' ), 100 );
+		add_filter( 'tribe_tickets_move_tickets_template_vars', array( $this, 'move_tickets_dialog_vars' ) );
+		add_filter( 'tribe_tickets_move_tickets_script_data', array( $this, 'move_tickets_dialog_data' ) );
+	}
+
+	public function move_tickets_dialog_vars( array $vars ) {
+		if ( ! $this->is_move_tickets_dialog() ) {
+			return $vars;
+		}
+
+		return array_merge( $vars, array(
+			'title'    => __( 'Move Ticket Types', 'event-tickets' ),
+			'mode'     => 'ticket_type_only',
+		) );
+	}
+
+	public function move_tickets_dialog_data( array $data ) {
+		if ( ! $this->is_move_tickets_dialog() ) {
+			return $data;
+		}
+
+		return array_merge( $data, array(
+			'ticket_type_id' =>
+				absint( $_GET[ 'ticket_type_id' ] ),
+			'src_post_id' =>
+				absint( $_GET[ 'post' ] ),
+			'mode' =>
+				'ticket_type_only'
```

Comment:
This array is formatted in a fairly non-standard way compared to what we do elsewhere. The last element is missing a trailing comma, too.